CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Forge
+
+#### Changed
+
+- Changed the code generated by `snforge_std`'s `#[test]` attribute
```

Comment:
Imo it doesn't tell much to a regular user, could we rephrase it?

Reply:
I think the only important part from the user standpoint is that it changed and that they need to update `snforge_std` (which we will have a separate warning for). Generated code itself is an implementation detail.